testutil/datagen/account_balance.go

```diff
@@ -15,6 +18,34 @@ func GenRandomAccount() *authtypes.BaseAccount {
 	return acc
 }
 
+func GenRandomAccountWithPrefix(prefix string) *authtypes.BaseAccount {
+	senderPrivKey := sec256k1.GenPrivKey()
+	acc := MustNewBaseAccountWithPrefix(senderPrivKey.PubKey().Address().Bytes(), senderPrivKey.PubKey(), 0, 0, prefix)
+	return acc
+}
```

Comment:
Can we use `sdk.MustBech32ifyAddressBytes` to convert a `sdk.AccAddress` to a string with a given prefix? This way we don't need the other functions introduced in this PR

Reply:
Not sure. We need to convert to a `BaseAccount`.

Reply:
I see. Would it be possible to get rid of the `BaseAccount` creation since we are only using `staker.Address` in the `staker` object ([ref](https://github.com/babylonlabs-io/babylon/pull/513/files#diff-131bdafa81adc9d991427ed518305f553a1077c162d190a0d923b0497f151ac6R168))?

Reply:
We have `Address` (string) `AccAddress` ([]byte), and `BaseAccount` (among others...)

Question would be, how to get a random `AccAddress`, i.e. the random (but valid) stream of bytes that represents a pubkey, and can be converted to an address string.

Reply:
Point is, that this `staker.Address` is _already_ the (bech32-formatted) address string with a given prefix. I can convert it to bytes and the bech32ify again with the new prefix, but... the current solution is better / clearer, IMO.

Reply:
Oh, I can use this: `senderPrivKey.PubKey().Address().Bytes()` I guess.

Reply:
> senderPrivKey.PubKey().Address().Bytes()

yes imo generating a random pk, deriving byte address, and then bech23ifying it with the prefix is most clean

Reply:
Done.